dotnetv3/Bedrock-runtime/Tests/ConverseToolScenarioTests.cs

```diff
@@ -0,0 +1,64 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using Amazon.BedrockRuntime;
+using ConverseToolScenario;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+
+namespace BedrockRuntimeTests;
+
+/// <summary>
+/// Tests for the Converse Tool Use example.
+/// </summary>
+public class ConverseToolScenarioTests
+{
+    private readonly BedrockActionsWrapper _bedrockActionsWrapper = null!;
+    private readonly WeatherTool _weatherTool = null!;
+    private readonly ILoggerFactory _loggerFactory;
+
+    /// <summary>
+    /// Constructor for the test class.
+    /// </summary>
+    public ConverseToolScenarioTests()
+    {
+
+        _loggerFactory = LoggerFactory.Create(builder =>
+        {
+            builder.AddConsole();
+        });
+
+        IServiceCollection services = new ServiceCollection(); // [1]
+
+        services.AddHttpClient();
+
+        IHttpClientFactory _httpClientFactory = services
+            .BuildServiceProvider()
+            .GetRequiredService<IHttpClientFactory>();
+
+        _bedrockActionsWrapper = new BedrockActionsWrapper(
```

Comment:
Set the region here as well, to make sure you can use Nova:
```
_bedrockActionsWrapper = new BedrockActionsWrapper(
    new AmazonBedrockRuntimeClient(RegionEndpoint.USEast1), new Logger<BedrockActionsWrapper>(_loggerFactory));
```

Reply:
Updated.